app/src/main/java/org/kinecosystem/kinit/view/earn/WebTaskTruexFragment.kt

```diff
@@ -16,6 +18,18 @@ import org.kinecosystem.kinit.viewmodel.earn.WebFragmentActions
 import org.kinecosystem.kinit.viewmodel.earn.WebTaskTruexViewModel
 
 class WebTaskTruexFragment : BaseFragment(), WebFragmentActions {
+    override fun openBrowser(url: String) {
+        val parsedUrl = Uri.parse(url)
+        val intent = Intent(Intent.ACTION_VIEW, parsedUrl)
+        context?.let {
+            if (parsedUrl == null || intent.resolveActivity(it.packageManager) == null) {
+                showErrorDialog()
+            } else {
+                startActivity(intent)
+            }
+        } ?: showErrorDialog()
```

Comment:
if you don't have context - you can't show the dialog - so no need for the elvis operator here

Reply:
pls remove this - then you can merge